### PR TITLE
adding metadata info to footer

### PR DESCRIPTION
**Pull request by callumtipper-IW**

### CODE REVIEW

File: client/index.html [hunk 1]
```diff
@@ -944,6 +944,19 @@ <h2 class="nhsuk-error-summary__title" id="error-summary-title">
       </div>
     </main>
   </div>
+  <footer role="contentinfo">
+    <div class="nhsuk-footer" id="nhsuk-footer">
+      <div class="nhsuk-width-container">
+        <h2 class="nhsuk-heading-l">Validator Packages:</h2>
```
Comment (mfjarvis) [inline client/index.html:950]: Hide header if `validatorPackages` is empty (i.e. user is unauthenticated)?

Comment (mfjarvis) [reply client/index.html:950]: Also "Validator Packages" probably isn't the right description as this now contains the EPS API version as well

Comment (mfjarvis) [reply client/index.html:950]: "Validator Packages" probably isn't the right description as this now contains the EPS API version as well

File: client/index.ts [hunk 2]
```diff
@@ -2394,6 +2397,53 @@ function setInitialState(mode: string, env: string, baseUrl: string, signRespons
   pageData.loggedIn = pageData.loggedIn || env.endsWith("-sandbox")
 }
 
+interface MetadataResponse {
+  capabilityStatement: {
+    extension: Array<ExtensionExtension<ExtensionExtension<StringExtension>>>
```
Comment (mfjarvis) [inline client/index.ts:2402]: Love this :ok_hand:

File: client/index.html (hunk 1 above)
Comment (dan-jones-iw) [reply client/index.html:950]: yea thought about it but then realised i could spend forever styling and improving

Comment (dan-jones-iw) [reply client/index.html:950]: its always gonna look dodgy happy to make this change tho

**Review by mfjarvis — APPROVED** [review]

File: client/index.ts [hunk 3]
```diff
@@ -192,6 +192,8 @@ const errorList: Array<{ message: string }> = undefined
 
 const payloads: Array<string> = []
 
+const validatorPackages: Array<{ name: string, version: string }> = []
```
Comment (mfjarvis) [inline client/index.ts:195]: This `{ name: string, version: string }` type is used in a few places. Could give it its own interface? `SoftwareVersion` or `ComponentVersion` or something

File: client/index.html (hunk 1 above)
Comment (mfjarvis) [reply client/index.html:950]: Fair, happy for you to ignore if you want

File: client/index.ts (hunk 3 above)
Comment (dan-jones-iw) [reply client/index.ts:195]: sounds good

**Review by mfjarvis — APPROVED** [review]

**Pull request merged**